src/components/results/career/modal/index.js

```diff
@@ -17,6 +18,8 @@ import style from "./style.scss";
 export default function CareerModal() {
   const [activeTab, setActiveTab] = useState(null);
   const career = useCareer();
+  const {inlineJobs, jobSource} = useOption("career")?.jobOptions || {};
```

Comment:
I think since it's already in an options hash, `jobOptions` can just be `job`.

options = {career: {job: {inline: true, source: "wendys"}}}

Reply:
OK changed jobOptions to jobs!